docs/rules/better-regex.md

```diff
@@ -1,11 +1,10 @@
 # Improve regexes by making them shorter, consistent, and safer
 
-<!-- Do not manually modify RULE_NOTICE part. Run: `npm run generate-rule-notices` -->
```

Comment:
@bmish I think it may be smart to have a notice like this.

Maybe:

```
<!-- end rule header -->
<!-- DO NOT EDIT ANYTHING ABOVE HERE -->
```

Reply:
I added those original warning comments back in. But they aren't managed by eslint-doc-generator as they are outside the managed area indicated by the `<!-- end rule header -->` comment.

Reply:
Filed an issue to consider making this part of the standard comment marker: https://github.com/bmish/eslint-doc-generator/issues/150